src/Bridges/Constraint/det.jl

```diff
@@ -98,7 +98,8 @@ function bridge_constraint(::Type{LogDetBridge{T}}, model,
 end
 
 MOI.supports_constraint(::Type{LogDetBridge{T}}, ::Type{<:Union{MOI.VectorOfVariables, MOI.VectorAffineFunction{T}}}, ::Type{MOI.LogDetConeTriangle}) where T = true
-added_constraint_types(::Type{LogDetBridge{T}}, ::Type{<:Union{MOI.VectorOfVariables, MOI.VectorAffineFunction{T}}}, ::Type{MOI.LogDetConeTriangle}) where T = [(MOI.VectorAffineFunction{T}, MOI.PositiveSemidefiniteConeTriangle), (MOI.VectorAffineFunction{T}, MOI.ExponentialCone), (MOI.ScalarAffineFunction{T}, MOI.LessThan{T})]
+MOIB.added_constrained_variable_types(::Type{<:LogDetBridge}) = Tuple{DataType}[]
```

Comment:
Is `Tuple{DataType}` really intended? That means just things like `(Int,)`? If so, why not just `DataType[]`? Maybe the return type should be specified in more detail in the doctoring.

Reply:
Even better, why not
```
Type{<:MOI.AbstractSet}[]
```

Reply:
`Tuple{DataType}` for constrained variables is for consistency with `Tuple{DataType, DataType}` for constraints. I didn't know `Type{<:MOI.AbstractSet}[]` was possible.

Reply:
`Type{<:MOI.AbstractSet}[]` seems to work

Reply:
There seems to be issues implementing the solution:
```julia
julia> d = Dict{Tuple{Type{<:Integer}}, Int}()
Dict{Tuple{Type{#s712} where #s712<:Integer},Int64} with 0 entries

julia> d[(Int,)] = 1
ERROR: StackOverflowError:
Stacktrace:
 [1] convert(::Type{Tuple{Type{#s712} where #s712<:Integer}}, ::Tuple{DataType}) at ./essentials.jl:275
 [2] setindex!(::Dict{Tuple{Type{#s712} where #s712<:Integer},Int64}, ::Int64, ::Tuple{DataType}) at ./dict.jl:373
 [3] setindex!(::Dict{Tuple{Type{#s712} where #s712<:Integer},Int64}, ::Int64, ::Tuple{DataType}) at ./dict.jl:377 (repeats 9994 times)
 [4] top-level scope at none:0
```
It would be interesting this figure this out since we use a lot of `DataType` in MOI and it would be more readable to use `Tuple{<:...}` indeed.

Reply:
Moving to a separate issue: https://github.com/JuliaOpt/MathOptInterface.jl/issues/783